packages/synapse-react-client/src/components/EntityFinder/EntityFinderModal.stories.ts

```diff
@@ -1,26 +1,18 @@
 import { Meta, StoryObj } from '@storybook/react'
-import { EntityFinderModal } from './EntityFinderModal'
+import { fn } from '@storybook/test'
 import {
-  SinglePane as EntityFinderSinglePane,
   DualPane as EntityFinderDualPane,
+  SinglePane as EntityFinderSinglePane,
 } from './EntityFinder.stories'
-import { fn } from '@storybook/test'
+import { EntityFinderModal } from './EntityFinderModal'
 
-const meta = {
+const meta: Meta<typeof EntityFinderModal> = {
   title: 'Synapse/EntityFinderModal',
   component: EntityFinderModal,
   args: {
     onConfirm: fn(),
     onCancel: fn(),
   },
-  argTypes: {
-    versionSelection: {
-      options: ['REQUIRED', 'DISABLED', 'TRACKED', 'UNTRACKED'],
-      control: {
-        type: 'select',
-      },
-    },
-  },
```

Comment:
Fixing types revealed this `argType` doesn't work--we have such a control for the `EntityFinder` story suite, so it's not important that it is perfect in the `EntityFinderModal` story suite.